lib/core/language-menu/language-menu.component.stories.ts

```diff
@@ -0,0 +1,84 @@
+/*!
+ * @license
+ * Copyright 2022 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { LanguageMenuComponent } from './language-menu.component';
+import { LanguageMenuModule } from './language-menu.module';
+import { LanguageService } from '../services/language.service';
+import { LanguageServiceMock } from '../mock/language.service.mock';
+import { CoreStoryModule } from '../testing/core.story.module';
+import { componentWrapperDecorator, Meta, moduleMetadata, Story } from '@storybook/angular';
+import { action } from '@storybook/addon-actions';
+import { CommonModule } from '@angular/common';
+import { MaterialModule } from '../material.module';
+import { TranslateModule } from '@ngx-translate/core';
+
+export default {
+    component: LanguageMenuComponent,
+    title: 'Core/Components/Language Menu/Language Menu',
+    decorators: [
+        moduleMetadata({
+            imports: [CoreStoryModule, CommonModule, MaterialModule, TranslateModule, LanguageMenuModule],
+            providers: [
+                { provide: LanguageService, useClass: LanguageServiceMock }
+            ]
+        })
+    ],
+    excludeStories: /.*Data$/
```

Comment:
do we need that?

Reply:
No actually, will remove that line in next push :)